swift-ci/master/amazon-linux/2/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM amazonlinux:2
 
 RUN yum install shadow-utils -y
 
-RUN groupadd -g 998 build-user && \
+RUN groupadd -g 1000 build-user && \
```

Comment:
I don't have any opinion on the group number, just curious why this change was made?

Reply:
I cannot remember why this was needed. It definitely was for CentOS when using the release image, but I don't know if it is still needed now that we're not. I'm guessing probably not? I don't suppose you remember @shahmishal / @DougGregor?

Reply:
I have no memory of this place

Reply:
I've reverted for now. We'll see if there's any errors pretty quick.